py/desiutil/test/test_io.py

```diff
@@ -8,8 +8,9 @@
 import unittest
 import sys
 import numpy as np
+from astropy.table import Table
 #import pdb
-from desiutil.io import yamlify, combine_dicts
+from desiutil import io
```

Comment:
This is dangerous, because it masks the Python 3 io library. https://docs.python.org/3/library/io.html